style-guide.md

```diff
@@ -21,51 +21,51 @@ export default () => (
 ✅ DO:
 
 ```js
-// Ejemplo
+// Exemple
 import React from "react"
 export default () => (
-  <div style={{ color: `purple`, fontSize: `72px` }}>Hello Gatsby!</div>
+  <div style={{ color: `purple`, fontSize: `72px` }}>Salut Gatsby!</div>
 )
 ```
 
-✅ ALSO OKAY:
+✅ OK:
 
 ```js
-// Ejemplo
+// Exemple
 import React from "react"
 export default () => (
-  <div style={{ color: `purple`, fontSize: `72px` }}>¡Hola Gatsby!</div>
+  <div style={{ color: `purple`, fontSize: `72px` }}>Salut Gatsby!</div>
```

Comment:
In French, there is a space before the question mark

```suggestion
  <div style={{ color: `morado`, fontSize: `72px` }}>Salut Gatsby !</div>
```

Reply:
For morado, wouldn't it be better to use violet?